cli/src/vote.rs

```diff
@@ -1760,32 +2267,54 @@ mod tests {
 
         let authorized_sig = authorized_keypair.sign_message(&[0u8]);
         let authorized_signer = format!("{}={}", authorized_keypair.pubkey(), authorized_sig);
-        let test_authorize_voter = test_commands.clone().get_matches_from(vec![
-            "test",
-            "vote-authorize-voter",
-            &pubkey_string,
-            &authorized_keypair.pubkey().to_string(),
-            &pubkey2_string,
-            "--blockhash",
-            &blockhash_string,
-            "--signer",
-            &authorized_signer,
-            "--signer",
-            &signer2,
-            "--fee-payer",
-            &pubkey2_string,
-            "--nonce",
-            &nonce_account.to_string(),
-            "--nonce-authority",
-            &pubkey2_string,
-        ]);
+        let test_authorize_voter = if with_bls {
+            test_commands.clone().get_matches_from(vec![
+                "test",
+                "vote-authorize-voter-with-bls",
```

Comment:
@samkim-crypto @zz-sol I'm a bit confused how to make things work here. I believe this one is for offline signing. In our case I'm doing BLS keypair derivation inside cli parser, so I always need the vote authority keypair passed in. How do we make offline signing work in this case? Is it:
1. do `vote-authorize-voter-with-bls` with `sign-only` to generate the whole message encrypted
2. So when this thing is called we should have BLS pubkey and PoP without the keypair online
Does that mean I should somehow pass in the BLS pubkey and PoP?